nrf_dm/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,51 @@ Changelog
 
 All the notable changes to this project are documented in this file.
 
+main
+****
+
+All the notable changes added to the |NCS| main are documented in this section.
+
+Added
+=====
+
+* Added experimental support for the nRF5340 network core:
+
+  * ``lib/nRF5340_CPUNET/soft_float/libnrf_dm.a``
+
+* Added experimental support for a reduced library for the nRF5340 application core:
+
+  * ``lib/nRF5340_CPUAPP/hard_float/libnrf_dm_calc.a``
+
+* Added experimental support for nRF52832:
+
+  * ``lib/nRF52832/hard_float/libnrf_dm.a``
+
+Changes
+=======
+
+* Added a quality indicator per tone.
+* Reduced the memory usage in dm_calc.
+* Changed internal timings which makes this release not compatible with previous releases.
+* Changed the handling of the reports.
+  :c:func:`nrf_dm_calc` and :c:func:`nrf_dm_high_precision_calc` now take the report as an input.
+  The report needs to be populated using :c:func:`nrf_dm_populate_report` before it can be used for :c:func:`nrf_dm_calc` and :c:func:`nrf_dm_high_precision_calc`.
+* Removed distance offset compensation of fixed user-configurable offsets.
+  This now needs to be done in the application.
+
+
+Migration information
+=====================
+* This release is not compatible with previous versions of nrf_dm.
+* The application now needs to provide memory for the report.
+* Use :c:func:`nrf_dm_populate_report` to load the raw data.
+  This needs to be done before passing the report to :c:func:`nrf_dm_calc` or :c:func:`nrf_dm_high_precision_calc`
+* :c:func:`nrf_dm_calc` and :c:func:`nrf_dm_high_precision_calc` now take the report as an input.
```

Comment:
take -> takes

Reply:
Take, since both functions take the report as input.